2025-05-28 00:09:40,967 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.6.1.158:44997'
2025-05-28 00:09:42,810 - distributed.worker - INFO -       Start worker at:     tcp://10.6.1.158:44267
2025-05-28 00:09:42,810 - distributed.worker - INFO -          Listening to:     tcp://10.6.1.158:44267
2025-05-28 00:09:42,810 - distributed.worker - INFO -           Worker name:             SLURMCluster-3
2025-05-28 00:09:42,810 - distributed.worker - INFO -          dashboard at:           10.6.1.158:33567
2025-05-28 00:09:42,810 - distributed.worker - INFO - Waiting to connect to: tcp://146.186.150.12:34667
2025-05-28 00:09:42,810 - distributed.worker - INFO - -------------------------------------------------
2025-05-28 00:09:42,810 - distributed.worker - INFO -               Threads:                          1
2025-05-28 00:09:42,810 - distributed.worker - INFO -                Memory:                 200.00 GiB
2025-05-28 00:09:42,810 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-4l7osmc8
2025-05-28 00:09:42,810 - distributed.worker - INFO - -------------------------------------------------
2025-05-28 00:09:42,838 - distributed.worker - INFO - Starting Worker plugin shuffle
2025-05-28 00:09:42,838 - distributed.worker - INFO -         Registered to: tcp://146.186.150.12:34667
2025-05-28 00:09:42,838 - distributed.worker - INFO - -------------------------------------------------
2025-05-28 00:09:42,839 - distributed.core - INFO - Starting established connection to tcp://146.186.150.12:34667
2025-05-28 01:33:11,108 - distributed.worker - ERROR - Compute Failed
Key:       model_vs-edf07d01-b225-4099-8ab5-756a246a10e0
State:     executing
Task:  <Task 'model_vs-edf07d01-b225-4099-8ab5-756a246a10e0' model_vs(...)>
Exception: 'NameError("name \'lon_min_sc\' is not defined")'
Traceback: '  File "/tmp/ipykernel_294052/4023677770.py", line 22, in model_vs\n'

slurmstepd: error: *** JOB 38802762 ON p-sc-2388 CANCELLED AT 2025-05-28T02:10:07 DUE TO TIME LIMIT ***
